README.md

```diff
@@ -0,0 +1,31 @@
+What is this?
+=============
+
+Frotz activity is a simple front end to the classic interactive fiction interpreter Frotz on the XO laptop.  Frotz activity is licensed under the GNU General Public License version 3 (see the file COPYING for details). 
+
+The actual interactive interpreter, Frotz, is licensed under the GNU General Public License version 2. You can obtain the source from here: http://frotz.homeunix.org/frotz/
```

Comment:
Link does not work.

Reply:
License of Frotz includes the _or later version_ wording.

Reply:
Done in c89585f